lib/vanagon/cli/list.rb

```diff
@@ -0,0 +1,68 @@
+require 'docopt'
+
+class Vanagon
+  class CLI
+    class List < Vanagon::CLI
+      DOCUMENTATION = <<~DOCOPT.freeze
+        Usage:
+          list [options]
+
+        Options:
+          -h, --help                       Display help
+          -c, --configdir DIRECTORY        Configuration directory [default: #{Dir.pwd}/configs]
+          -l, --platforms                  Display a list of platforms
+          -r, --projects                   Display a list of projects
+          -s, --use-spaces                 Displays the list as space separated
+      DOCOPT
+
+      def parse(argv)
+        Docopt.docopt(DOCUMENTATION, { argv: argv })
+      rescue Docopt::Exit => e
+        puts e.message
+        exit 1
+      end
+
+      def output(list, use_spaces)
+        return list.join(' ') if use_spaces
+        return list
+      end
+
+      def run(options) # rubocop:disable Metrics/AbcSize
+        platform_list = Dir.children(File.join(options[:configdir], '/platforms')).map do |platform|
```

Comment:
should be: `File.join(options[:configdir], 'platforms')`

The '/' is provided by #join

Reply:
ah yes... and likewise for `/projects'